### PR TITLE
doc: Move Thingy:91 and Thingy:91 x under application development

### PR DESCRIPTION
**Pull request by divipillai**
Moved Thingy:91 and Thingy:91 x under application development.

NCSDK-29643

### CODE REVIEW

File: doc/_zoomin/ncs.tags.yml [hunk 1]
```diff
@@ -198,6 +194,10 @@ mapping_topics:
                                                          "development-kits",
                                                          "prototyping-platforms"]
   - nrf/app_dev/device_guides/nrf91/nrf9160_external_flash.html: ["nrf9160", "development-kits"]
+  - nrf/app_dev/device_guides/thingy91/*.html: ["thingy91", "prototyping-platforms"]
+  - nrf/app_dev/device_guides/nrf91/thingy91_updating_fw_programmer.html: ["nrf9160"]
+  - nrf/app_dev/device_guides/thingy91x/*.html: ["thingy91x", "prototyping-platforms"]
```
Comment (umapraseeda) [inline doc/_zoomin/ncs.tags.yml:199]: nrf91 series tag could be added

**Review by umapraseeda — CHANGES_REQUESTED** [review]

File: doc/nrf/app_dev/device_guides/thingy91x/thingy91x_features.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
+.. _thingy91x_features:
+
+Features of Thingy:91 X
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Nordic Thingy:91 X is an advanced version of the Thingy:91.
```
Comment (bjornspock) [inline doc/nrf/app_dev/device_guides/thingy91x/thingy91x_features.rst:10]: Thingy:91 and THingy:91 X are two destinct products, so I dont think we should state that the Thingy:91 X is an advanced version of the Thingy:91. Suggest replacing the sentence with the following:

"Nordic Thingy:91 X is a battery-operated prototyping platform for cellular IoT. "

**Review by umapraseeda — APPROVED** [review]

**Review by martiles — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]
Checked the doc preview and it looks OK!

**Comment by divipillai** [comment]
The compliance error is because of the image size of existing images. I think we could override this compliance check?

**Review by FrancescoSer — APPROVED** [review]

**Pull request merged**